scripts/utils/setup-roles.ts

```diff
@@ -113,33 +131,23 @@ export async function main (
     console.log(`feeController has transferred ownership to address: ${adminMultiSig.address}`);
 
     if (CALL_WHITELIST_ADDRESS) {
-    // set CallWhiteList admin
-    const whitelist = await ethers.getContractAt("CallWhitelist", CALL_WHITELIST_ADDRESS);
-    const updateWhitelistAdmin = await whitelist.transferOwnership(adminMultiSig.address);
-    await updateWhitelistAdmin.wait();
+        // set CallWhiteList admin
+        const whitelist = await ethers.getContractAt("CallWhitelist", CALL_WHITELIST_ADDRESS);
+        const updateWhitelistAdmin = await whitelist.transferOwnership(adminMultiSig.address);
+        await updateWhitelistAdmin.wait();
     }
 
     console.log(`whitelist has transferred ownership to address: ${adminMultiSig.address}`);
 
     if (PUNK_ROUTER_ADDRESS) {
-    // set PunkRouter admin
-    const punkRouter = await ethers.getContractAt("PunkRouter", PUNK_ROUTER_ADDRESS);
-    const updatepunkRouterAdmin = await punkRouter.transferOwnership(adminMultiSig.address);
-    await updatepunkRouterAdmin.wait();
+        // set PunkRouter admin
+        const punkRouter = await ethers.getContractAt("PunkRouter", PUNK_ROUTER_ADDRESS);
+        const updatepunkRouterAdmin = await punkRouter.transferOwnership(adminMultiSig.address);
+        await updatepunkRouterAdmin.wait();
     }
 
     console.log(`punkRouter has transferred ownership to address: ${adminMultiSig.address}`);
     console.log(SECTION_SEPARATOR);
     console.log("Transferred all ownership.\n");
-}
 
-// We recommend this pattern to be able to use async/await everywhere
```

Comment:
Why remove this? This and the fact that the arguments above are set as `Contract` type (instead of the string addresses they previously were) make it impossible to run `setup-roles.ts` as an independent script. This is a worse outcome than before so I think we need to undo this. My recommendation is to keep the same type signature of the function (all parameters are `string` types and the values are addresses), just remove the default values.

Reply:
Actually, forget this.....the bottom `main` function doesn't even pass the right variables. 

Maybe we can merge as-is now, and have `setup-roles` in the future, if not provided arguments, read and initialize the contracts from the addresses JSON file we described for the programmatic-verification task.